packages/ui-shared/src/components/ButtonLinkBox/ButtonLinkBox.docz.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+
+export const BackgroundWrapper: React.FC = ({ children }) => (
```

Comment:
```suggestion
export const DemoBackgroundWrapper: React.FC = ({ children }) => (
```
Чтобы смотря на доку не было желания использовать компонент BackgroundWrapper, которого в либе нет.